docs/src/jsx/commands/GLText.js

```diff
@@ -171,7 +171,7 @@ function Example() {
           phi: oldState.phi,
         }))
       }>
-      <GLText>{matrix.toMarkers()}</GLText>
+      <GLText alphabet={ALPHABET}>{matrix.toMarkers()}</GLText>
```

Comment:
where is `ALPHABET` defined?

Reply:
It's in that same file, line24:
https://github.com/cruise-automation/webviz/blob/0d445c5e315296af47fb9376068d2fe7ac999f0f/docs/src/jsx/commands/GLText.js#L24